core/pta/device.c

```diff
@@ -21,9 +21,15 @@
 #define PTA_NAME "device.pta"
 
 static void add_ta(uint32_t flags, const TEE_UUID *uuid, uint8_t *buf,
-		   uint32_t blen, uint32_t *pos)
+		   uint32_t blen, uint32_t *pos, uint32_t rflags)
 {
-	if (flags & TA_FLAG_DEVICE_ENUM) {
+	if ((flags & TA_FLAG_DEVICE_ENUM) &&
+	    (flags & TA_FLAG_DEVICE_ENUM_SUPP)) {
+		EMSG("pta: skipping ta %pUl, broken enum flags\n", uuid);
```

Comment:
I have 4 comments to make on this line ;-)
1. You could use PTA_NAME
2. s/ta/TA/
3. "broken": I prefer "inconsistent"
4. `\n` not needed

Therefore:
```C
EMSG(PTA_NAME ": skipping TA %pUl, inconsistent flags");
```